primeqa/mrc/trainers/mrc.py

```diff
@@ -27,13 +35,22 @@ def __init__(self, *args, eval_examples=None, eval_dataset=None, post_process_fu
             post_process_function:  Function to create predictions from model outputs.
             **kwargs: Keyword arguments for super-class constructor.
         """
-        kwargs['args'].label_names = ['start_positions', 'end_positions']
+        # FIXME: Work around for a bug in Transformers (4.24.0)
+        # Reference: https://github.com/huggingface/transformers/issues/20105
+        # -------- START OF FIX --------
+        if transformers.__version__ == "4.24.0" and "args" not in kwargs:
```

Comment:
This solution looks nice and clean but it needs to be modified for the run_mrc.py case where the args are already there.

Reply:
@seirasto @kpfadnis : who is fixing this ? This is a pre-req for Tapex and hence I need both your sign-offs from services and primeqa mrc side, before I can approve/merge it.

Reply:
@seirasto Made changes as per our conversation. Please take a look when you have a minute.

@jdpsen Once @seirasto reviews, you might can do full review for the PR.

Reply:
LGTM! Thanks @kpfadnis